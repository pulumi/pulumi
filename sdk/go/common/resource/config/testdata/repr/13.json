{"my:servers":[{"host":"example","port":80,"token":{"secure":"securevalue"}}]}