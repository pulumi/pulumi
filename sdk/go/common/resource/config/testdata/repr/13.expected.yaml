decrypt:
  my:servers: '[{"host":"example","port":80,"token":{"secure":"securevalue"}}]'
paths:
  my:servers:
    value:
    - host: example
      port: 80
      token:
        secure: securevalue
    string: '[{"host":"example","port":80,"token":{"secure":"securevalue"}}]'
    redacted: '[{"host":"example","port":80,"token":"[secret]"}]'
    object: HRAADltdaW50ZXJmYWNlIHt9/4MCAQL/hAABEAAA/5j/hCgAARdtYXBbc3RyaW5nXWludGVyZmFjZSB7ff+BBAEC/4IAAQwBEAAAbP+CaQADBXRva2VuF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4IfAAEGc2VjdXJlBnN0cmluZwwNAAtzZWN1cmV2YWx1ZQRob3N0BnN0cmluZwwJAAdleGFtcGxlBHBvcnQFaW50NjQEAwD/oA==
    secure: true
    isObject: true
    secureValues:
    - securevalue
  my:servers[0]:
    value:
      host: example
      port: 80
      token:
        secure: securevalue
    string: '{"host":"example","port":80,"token":{"secure":"securevalue"}}'
    redacted: '{"host":"example","port":80,"token":"[secret]"}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAABs/4JpAAMEaG9zdAZzdHJpbmcMCQAHZXhhbXBsZQRwb3J0BWludDY0BAMA/6AFdG9rZW4XbWFwW3N0cmluZ11pbnRlcmZhY2Uge33/gh8AAQZzZWN1cmUGc3RyaW5nDA0AC3NlY3VyZXZhbHVl
    secure: true
    isObject: true
    secureValues:
    - securevalue
  my:servers[0].host:
    value: example
    string: example
    redacted: example
    object: FBAABnN0cmluZwwJAAdleGFtcGxl
    secure: false
    isObject: false
  my:servers[0].port:
    value: "80"
    string: "80"
    redacted: "80"
    object: DxAABnN0cmluZwwEAAI4MA==
    secure: false
    isObject: false
  my:servers[0].token:
    value:
      secure: securevalue
    string: securevalue
    redacted: '[secret]'
    object: GBAABnN0cmluZwwNAAtzZWN1cmV2YWx1ZQ==
    secure: true
    isObject: false
    secureValues:
    - securevalue
